### PR TITLE
Add Ag cohorts analysis 20240924

### LINKED ISSUE #32: Ag cohorts analysis 20240924
New cohorts analysis has been released. Includes some notable changes:

* gcx1 -> bissau
* fix error regarding Teso

Applies up to Ag3.13.

### PR DESCRIPTION
**Pull request by ahernank**
* Resolves #32

### CODE REVIEW

**Comment by ahernank** [comment]
@cclarkson @alimanfoo - I've had a go at drafting a short explanation here as we've had a query about this from the support mailbox and would be nice to point them towards the vobs-updates note, I have included that this change is up to Ag3.11 as this is the latest public release available. 

Please feel free to edit directly as you see fit.

File: _posts/2024-10-02-ag3-cohorts-v20240924.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+layout: post
+title:  "Ag3 cohorts analysis version 20240924"
+tags: data
+---
+
+A new cohorts analysis version `20240924` has been released for the
+Ag3 data resource. This is now the default cohorts analysis version
+when using the `malariagen_data` [Ag3
+API](https://malariagen.github.io/malariagen-data-python/latest/Ag3.html). This
+cohorts analysis is available for datasets up to and including Ag3.11.
```
Comment (alimanfoo) [inline _posts/2024-10-02-ag3-cohorts-v20240924.md:11]: I have been following the approach where we declare up to the release that will be covered by this cohorts analysis, even if the release is not public yet. So I'd suggest:
```suggestion
cohorts analysis will be available for datasets up to and including Ag3.13.
```

File: _posts/2024-10-02-ag3-cohorts-v20240924.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+This new version introduces some key changes: 
+
+- Samples previously labeled as `gcx1` in the `taxon` field have been renamed to `bissau`:
+    - `gcx` (`gambiae complex cryptic taxa`) labels serve as placeholders for groups outside our usual taxonomic assignment 
+    - Following [Caputo et al. (2024)](https://malariagen.github.io/vobs-updates/2024/09/10/caputo.html), the `gcx1` group has been renamed to `Bissau molecular form`
+    - 291 samples previously assigned as `gcx1`, are now labeled as `bissau`.
+    - 5 previously `unassigned` samples are also relabeled as `bissau`.
+    - Cohort names have been updated, e.g. `GM-M_gcx1_2019` is now `GM-M_biss_2019`
+
+- 36 `unassigned` samples have been reclassified as: 32 `melas`, 2`gambiae`, 1 `fontenillei`, 1 `arabiensis`.
```
Comment (alimanfoo) [inline _posts/2024-10-02-ag3-cohorts-v20240924.md:37]:
```suggestion
- 36 previously `unassigned` samples have been reclassified as: 32 `melas`, 2`gambiae`, 1 `fontenillei`, 1 `arabiensis`.
```

**Review by alimanfoo — COMMENTED** [review]
Looks good, thanks @ahernank 🙏🏻 

Couple of small suggestions...

**Comment by ahernank** [comment]
Merging shortly, unless there are any additional comments.

**Pull request merged**